main.js

```diff
@@ -0,0 +1,16 @@
+const speed = document.querySelector(".speed");
+const bar = speed.querySelector(".speed-bar");
+const video = document.querySelector(".flex");
+const min = 0.4;
+const max = 4;
+
+function boostVideo(e) {
+  const y = e.pageY - this.offsetTop;
+  const percent = y / this.offsetHeight;
+  const height = Math.round(percent * 100) + "%";
```

Comment:
You may use ES6 string for such cases,  I mean in case you combine data and any symbols